### ISSUE TITLE
Type inference doesn't work with IDE

### ISSUE THREAD

**Issue by JohnReedLOL** [deanwampler/spark-scala-tutorial#20]
Normally when I clone an sbt project from GitHub into IntelliJ, type inference works. 

I did `File > New > Project from Version Control > GitHub`, cloned it, and type inference does not work.

**Comment by deanwampler** [comment]
I don't know what you mean by "type inference doesn't work". Does the cloned project compile without error? If there are errors, perhaps the dependencies didn't get brought over successfully.

**Comment by JohnReedLOL** [comment]
Pressing "Ctrl-Q" in IntelliJ is supposed to bring up the type but it doesn't. The project compiles and runs with sbt.

**Comment by deanwampler** [comment]
I don't really know why that happens. What file?

**Comment by JohnReedLOL** [comment]
It's not a text file - the entire IDE type inference system was screwey for spark-scala-tutorial and only spark-scala-tutorial. For example, `val sc` of type "SparkContext" was being type inferred to "Any".

**Comment by JohnReedLOL** [comment]
Watch this video...

https://youtu.be/PWBVe-ISiJ8

Basically normally when you clone a repo into IntelliJ, type inference works. Ctr-Q pulls up the type. Hover over pulls up the type. 

What happened was when I imported _spark-scala-tutorial_ into IntelliJ and followed the exact same steps as I did for _hello-slick_, type inference did not work in any text file. For example, `val sc` of type "SparkContext" was being type inferred to "Any".

Unfortunately, when I got around to replicating it, everything worked.
